independent-projects/resteasy-reactive/pom.xml

```diff
@@ -97,6 +98,15 @@
                 <type>pom</type>
             </dependency>
 
+            <!-- Mockito BOM -->
+            <dependency>
+                <groupId>org.mockito</groupId>
+                <artifactId>mockito-bom</artifactId>
+                <version>${mockito.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+
```

Comment:
Just want to double check with @geoand and @gsmet that they're OK with this dependency being added.

Reply:
Do we really need the BOM? Isn't adding the mockito test dependency enough?

Reply:
🤷

Reply:
actually no, let me remove it. I'll try tu run the ci on the fork and then rebase